geomesa-kafka/geomesa-kafka-confluent/src/test/scala/org/locationtech/geomesa/kafka/confluent/SchemaParserTest.scala

```diff
@@ -0,0 +1,369 @@
+/***********************************************************************
+ * Copyright (c) 2013-2022 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.kafka.confluent
+
+import org.apache.avro.Schema
+import org.apache.avro.generic.GenericData
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.locationtech.geomesa.utils.text.WKBUtils
+import org.locationtech.jts.geom._
+import org.locationtech.jts.geom.impl.CoordinateArraySequenceFactory
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+import java.nio.ByteBuffer
+import java.util.Date
+
+@RunWith(classOf[JUnitRunner])
+class SchemaParserTest extends Specification {
```

Comment:
this class was copied from AvroSimpleFeatureTypeParserTest and probably doesn't need much/any review